client/azure_client_set.go

```diff
@@ -152,211 +129,108 @@ func NewAzureClientSet(config AzureClientSetConfig) (*AzureClientSet, error) {
 	return clientSet, nil
 }
 
-func newDeploymentsClient(config *clientConfig) (*resources.DeploymentsClient, error) {
-	c := resources.NewDeploymentsClientWithBaseURI(config.resourceManagerEndpoint, config.subscriptionID)
-	c.Authorizer = autorest.NewBearerAuthorizer(config.servicePrincipalToken)
-	c.RetryAttempts = 1
```

Comment:
I removed this line from all the clients. I don't think it's necessary anymore?

Reply:
Can I get a thumbs up on this comment @giantswarm/team-celestial ?

Reply:
Shouldn't be a problem I think.